### PR TITLE
[WIP] Scrub PII from website feedback using Scrubadub library

### PR DESCRIPTION
**Pull request by lbeaufort**
Connected to https://github.com/18F/fec-cms/issues/1583

Works on local [(Example)](https://github.com/18F/FEC/issues/4694) after running `pip install scrubadub`

### CODE REVIEW

File: fec/data/views.py [hunk 1]
```diff
@@ -466,8 +467,10 @@ def feedback(request):
                         request.META.get('HTTP_REFERER'),
                         request.META['HTTP_USER_AGENT'])
 
+            scrubbed=scrubadub.clean(body)
```
Comment (xtine) [inline fec/data/views.py]: Need spaces between `=`

**Comment by xtine** [comment]
You need to add the library to `requirements.txt`.

**Comment by lbeaufort** [comment]
@xtine thanks for your help! That what I was going to try next :) Did I add it in the best place? I also updated the issue with notes, but I'll put them here. Going to pause on this one until next sprint.


@xtine:

[scrubadub](http://scrubadub.readthedocs.io/en/stable/) currently supports removing:

Names (proper nouns) via textblob
Email addresses
URLs
Phone numbers via phonenumbers
username / password combinations
Skype usernames
Social security numbers

These [advanced techniques](http://scrubadub.readthedocs.io/en/stable/advanced_usage.html#advanced-usage) allow users to fine-tune the manner in which scrubadub cleans dirty dirty text.

**Comment by lbeaufort** [comment]
Stale PR

**Pull request closed without merging**